freyja/1.3.7/Dockerfile

```diff
@@ -0,0 +1,58 @@
+FROM mambaorg/micromamba:0.22.0 as app
+
+# Version arguments
+# ARG variables only persist during build time
+ARG FREYJA_SOFTWARE_VERSION="1.3.7"
+
+# build and run as root users since micromamba image has 'mambauser' set as the $USER
+USER root
+# set workdir to default for building; set to /data at the end
+WORKDIR /
+
+LABEL base.image="mambaorg/micromamba:0.22.0"
+LABEL dockerfile.version="2"
```

Comment:
please change dockerfile.version down to 1 since this is a new version of freyja